include/sunlinsol/sunlinsol_lapackdense.h

```diff
@@ -56,8 +56,8 @@ SUNDIALS_EXPORT SUNLinearSolver SUNLinSol_LapackDense(N_Vector y,
 SUNDIALS_EXPORT SUNLinearSolver_Type SUNLinSolGetType_LapackDense(SUNLinearSolver S);
 SUNDIALS_EXPORT SUNLinearSolver_ID SUNLinSolGetID_LapackDense(SUNLinearSolver S);
 SUNDIALS_EXPORT int SUNLinSolInitialize_LapackDense(SUNLinearSolver S);
-SUNDIALS_EXPORT int SUNLinSolSetup_LapackDense(SUNLinearSolver S, SUNMatrix A);
-SUNDIALS_EXPORT int SUNLinSolSolve_LapackDense(SUNLinearSolver S, SUNMatrix A,
+SUNDIALS_EXPORT SUNLsStatus SUNLinSolSetup_LapackDense(SUNLinearSolver S, SUNMatrix A);
+SUNDIALS_EXPORT SUNLsStatus SUNLinSolSolve_LapackDense(SUNLinearSolver S, SUNMatrix A,
                                                N_Vector x, N_Vector b, realtype tol);
```

Comment:
```suggestion
SUNDIALS_EXPORT SUNLsStatus SUNLinSolSolve_LapackDense(SUNLinearSolver S, SUNMatrix A,
                                                       N_Vector x, N_Vector b, realtype tol);
```

Reply:
If we can agree to run clang-format on all of the source immediately after this PR, then I think we can skip formatting suggestions which may help cut down on noise in the review. 

Personally, after looking over many many lines of code, I think it would be very nice to finally run the formatter on everything as the formatting style is more varied throughout the codebase than we might have expected.

Reply:
That would be good with me and would help streamline reviewing for sure.

Reply:
Agreed.  But, as you note, we should do a format AFTER this PR and not part  of it.  :)

Reply:
I wish I'd read this thread before suggesting formatting changes over the last hour.  I'll stop those now.

Reply:
I am verry happy about this decision